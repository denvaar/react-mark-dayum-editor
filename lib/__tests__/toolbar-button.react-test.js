import React from 'react';
import renderer from 'react-test-renderer';

import ToolbarButton from '../components/toolbar-button';


it('renders correctly', () => {
  const tree = renderer.create(
    <ToolbarButton tooltip={"tooltip text"} callback={null} icon={"fake-icon"} />
  ).toJSON();

  expect(tree).toMatchSnapshot();
});
