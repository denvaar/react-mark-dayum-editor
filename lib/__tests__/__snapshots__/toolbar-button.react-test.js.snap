// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<button
  className="md-toolbar-button"
  onClick={null}
  title="tooltip text"
  type="button"
/>
`;
